internal/git/cli.go

```diff
@@ -82,6 +86,34 @@ func (g *GitCLI) GlobalConfigPath() (string, error) {
 	return "", fmt.Errorf("unable to locate config file '%s'", filepath.Join("$HOME", ".gitconfig"))
 }
 
+func (g *GitCLI) runMerge(args ...string) (string, error) { // this function is implemented similar to the 'run' function below
+	c := exec.CommandContext(g.ctx, g.mergeBinary, args...)
+	c.Dir = g.cwd
+	c.Env = envMapToEntries(g.env)
+
+	if g.log {
+		fmt.Println(c.String())
+	}
+
+	if !g.quiet {
+		c.Stderr = os.Stderr
+	}
+
+	var stdout = bytes.Buffer{}
+	c.Stdout = &stdout
+	err := c.Run()
+	if e := (&exec.ExitError{}); err != nil && errors.As(err, &e) {
+		core.Debug("%d", e.ExitCode())
+		return stdout.String(), err
+	} else if err != nil {
+		core.Debug("126")
+	} else {
+		core.Debug("0")
+	}
```

Comment:
The process' stderr is completely ignored here. Do you mind logging that somehow as well? Maybe use `cmd.CombinedOutput()` instead of `cmd.Run()`?
Also, I think it would be good to provide some context with `merge` binary's exit code so that the log doesn't just show a number, e.g. `merge command exited with status 123`.

Reply:
> The process' stderr is completely ignored here. Do you mind logging that somehow as well? Maybe use cmd.CombinedOutput() instead of cmd.Run()?

In the `runMerge` function stderr is being assigned to `os.Stderr`.

> Also, I think it would be good to provide some context with merge binary's exit code so that the log doesn't just show a number, e.g. merge command exited with status 123.

Ok, agreed!